ci/cirrus_general_ci.yml

```diff
@@ -38,6 +38,46 @@ linux_aarch64_test_task:
     python dev.py test
 
 
+musllinux_amd64_test_task:
+  container:
+    image: alpine
+    cpu: 8
+    memory: 32G
+
+  env:
+    PATH: $PWD:$PATH
+
+  pip_cache:
+    folder: ~/.cache/pip
+    populate_script: |
```

Comment:
The docs say this is optional (https://cirrus-ci.org/guide/writing-tasks/#cache-instruction), and it didn't tell me why it's helpful to repeat the same install commands as in `setup_script`. Just to understand, why is this useful?

Reply:
The reason the cache could be important is to prevent numpy having to be rebuilt everytime. I think I can improve a bit on the caching. However, it's not clear to me from that link whether or not it's necessary to repeat the same pip install commands.

Reply:
My understanding was that that was only to pre-populate the cache, so I'd expect it to work without.